pallets/pallet-bonded-coins/src/curves/lmsr.rs

```diff
@@ -1,3 +1,28 @@
+/// Implementation of the Logarithmic Market Scoring Rule (LMSR) bonding curve.
+/// The cost of purchasing an amount of assets from the market is determined using the liquidity parameter of the
+/// LMSR model and the current supply of assets in the market.
+/// The LMSR bonding curve is defined by the following equation:
+///
+/// C(s) = m * ln(Σ(e^(s_i/m))),
+///
+/// where:
+/// - `s` is the supply of assets,
+/// - `m` is the liquidity parameter of the LMSR model,
+/// - `s_i` is the current supply of the assets.
+///
+/// `C(s)` represents the accumulated cost of purchasing/selling assets up to the current supply `s`.
+/// To calculate the incremental cost of purchasing the assets, use the formula:
+/// `C(s) - C(s*)`, where `s*` is the supply of assets in the market before the purchase.
+///
+///
+/// Components
```

Comment:
I don't think it's necessary to list these components here. They will just be exported in the doc of they are public. This is dangerous as it risks (almost certainly) to get out-of-date if we change anything in this module. I would simply remove this list here.

Reply:
Section droped in: [2ae4290](https://github.com/KILTprotocol/kilt-node/pull/801/commits/2ae429067b29b8adcd5200855c7b3e4dcfa0abe2)